docs/content/en/docs/Getting started/recovery.md

```diff
@@ -27,14 +27,16 @@ elemental upgrade --recovery
 It also supports to specify docker images directly:
 
 ```bash
-elemental upgrade --recovery --docker-image <image>
+elemental upgrade --recovery --recovery-system.uri <image-uri>
 ```
 
+Where `<image-uri>` can be an opaque URI of `docker` scheme (e.g. `docker:registry.org/some/image:tag`). Elemental supports `docker`, `channel` (refers to a Luet package from a repository), `dir` (refers to a local directory) and `file` (refers to a local file).
```

Comment:
this looks should go somewhere else, as it refers generally on how to address to sources on elemental side of things. Maybe (just a spare idea, feel free to ignore) should go into its own page (with some small examples) under Reference, and link to it in the relevant parts?

Reply:
indeed this could be omitted here and link to some reference documentation instead.